.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [18.x, 19.x]
```

Comment:
I'm currently working in a company that has node 16 or even 14. Could you think we can add support for it or only support current LTS + upper version?

Reply:
node 14 doesn't expose the webCrypto module that's required by the wasm module so i left it out, but yeah, including node 16 in the build matrix is ok